<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-06-13 16:32:02.911 ***</b>
=ERROR REPORT==== 13-Jun-2024::16:32:02.910847 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.243974.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1805}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1766}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-06-13 16:32:02.911 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:32:02.911356 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.243976.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2254)
    ancestors: [<0.243974.0>]
    message_queue_len: 0
    messages: []
    links: [<0.243974.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-06-13 16:32:03.720 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:32:03.720804 ===
'global' at node test_server@ce364c7721ac requested disconnect from node 'gen_server_SUITE-multicall_remote-583138-11794@ce364c7721ac' in order to prevent overlapping partitions

</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-06-13 16:32:43.251 ***</b>
=ERROR REPORT==== 13-Jun-2024::16:32:43.251356 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248326.0&gt;,
                               {#Ref&lt;0.1026865991.1650196481.250716&gt;,1006742,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.1026865991.1650196481.250716&gt;,1006742,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-06-13 16:32:43.251 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:32:43.251800 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248347.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1026865991.1650196481.250716&gt;,1006742,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [&lt;0.248326.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 4751
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-06-13 16:37:21.820 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:37:21.820183 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254085.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1026865991.1651769345.52216&gt;,6550,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1522)
    ancestors: [&lt;0.254083.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28124
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-13 16:37:31.054 ***</b>
=ERROR REPORT==== 13-Jun-2024::16:37:31.054558 ===
** Generic server <0.255358.0> terminating 
** Last message in was {nodedown,'peer_down_crash-592962-11794@ce364c7721ac'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-592962-11794",
                              peer_down => crash},
                            'peer_down_crash-592962-11794@ce364c7721ac',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,48,47,98,105,110,
                             47,101,114,108],
                            ["-sname","peer_down_crash-592962-11794",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQGNlMzY0Yzc3MjFhYwAD5X4AAAAAZmsAgg=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-592962-11794@ce364c7721ac'}},
                            {<0.255353.0>,#Ref<0.1026865991.1651769345.67206>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-592962-11794@ce364c7721ac'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-06-13 16:37:31.055 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:37:31.055114 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255358.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-592962-11794@ce364c7721ac'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2304)
    ancestors: [<0.255353.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11253
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-13 16:37:31.861 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:30.859783 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-359716-11794@ce364c7721ac' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-13 16:37:32.585 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:32.584828 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-359556-11794@ce364c7721ac'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-06-13 16:37:32.621 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:32.621733 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-359620-11794@ce364c7721ac'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-06-13 16:37:48.309 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:47.307503 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-359876-11794@ce364c7721ac' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-06-13 16:37:48.594 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:48.594844 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-593762-11794@ce364c7721ac'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-06-13 16:37:48.757 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:48.757759 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-398021-11794@ce364c7721ac'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:50.926 ***</b>
=ERROR REPORT==== 13-Jun-2024::16:37:50.926526 ===
** Generic server <0.255728.0> terminating 
** Last message in was {'EXIT',<0.255710.0>,
                               {#Ref<0.1026865991.1645215746.20106>,1521582,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-360228-11794",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-360228-11794@ce364c7721ac',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,48,47,98,105,110,47,101,114,108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-360228-11794",
                                     "-user","peer"],
                                    #Port<0.3120>,undefined,<<>>,running,
                                    {<0.255710.0>,
                                     #Ref<0.1026865991.1644691459.146029>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1026865991.1645215746.20106>,1521582,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:50.927 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:37:50.927616 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255728.0>
    registered_name: []
    exception exit: {#Ref<0.1026865991.1645215746.20106>,1521582,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.255710.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3120>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 13666
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:51.521 ***</b>
=ERROR REPORT==== 13-Jun-2024::16:37:51.521455 ===
** Generic server <0.255756.0> terminating 
** Last message in was {'EXIT',<0.255711.0>,
                           {#Ref<0.1026865991.1645215746.20134>,2115285,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "JGVKRUXARHZFSGFRMUAG","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ce364c7721ac.2024-06-13_16.21.53/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-360324-11794",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ce364c7721ac.2024-06-13_16.21.53/make_test_dir.stdlib_test.logs/run.2024-06-13_16.22.36/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-360324-11794",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-360324-11794@ce364c7721ac',
                                    "/usr/local/lib/erlang-26.2.5/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-360324-11794",
                                     "-setcookie","JGVKRUXARHZFSGFRMUAG",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ce364c7721ac.2024-06-13_16.21.53/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-360324-11794",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ce364c7721ac.2024-06-13_16.21.53/make_test_dir.stdlib_test.logs/run.2024-06-13_16.22.36/log_private/",
                                     "-user","peer"],
                                    #Port<0.3145>,undefined,<<>>,running,
                                    {<0.255711.0>,
                                     #Ref<0.1026865991.1645215746.22946>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1026865991.1645215746.20134>,2115285,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:51.522 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:37:51.522323 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255756.0>
    registered_name: []
    exception exit: {#Ref<0.1026865991.1645215746.20134>,2115285,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.255711.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3145>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 17731
    stack_size: 29
    reductions: 22344
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:51.871 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:50.870313 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-360292-11794@ce364c7721ac' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:51.995 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:51.995762 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-404227-11794@ce364c7721ac'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:52.271 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:52.271788 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-360356-11794@ce364c7721ac'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-06-13 16:37:52.410 ***</b>
=WARNING REPORT==== 13-Jun-2024::16:37:52.410731 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-360196-11794@ce364c7721ac'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-13 16:48:49.220 ***</b>
=ERROR REPORT==== 13-Jun-2024::16:48:49.220099 ===
** Generic server <0.302444.0> terminating 
** Last message in was {'EXIT',<0.302441.0>,
                               {#Ref<0.1026865991.1686372353.206368>,350851,
                                [{child1,<0.302450.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302444.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302450.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459137],
                            0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1026865991.1686372353.206368>,350851,
    [{child1,<0.302450.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-13 16:48:49.220 ***</b>
=ERROR REPORT==== 13-Jun-2024::16:48:49.220103 ===
** Generic server <0.302443.0> terminating 
** Last message in was {'EXIT',<0.302441.0>,
                               {#Ref<0.1026865991.1686372353.206368>,350851,
                                [{child1,<0.302450.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302443.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302449.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459137],
                            0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1026865991.1686372353.206368>,350851,
    [{child1,<0.302450.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-13 16:48:49.220 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:48:49.220500 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302444.0>
    registered_name: []
    exception exit: {#Ref<0.1026865991.1686372353.206368>,350851,
                     [{child1,<0.302450.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302441.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11937
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-06-13 16:48:49.220 ***</b>
=CRASH REPORT==== 13-Jun-2024::16:48:49.220646 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302443.0>
    registered_name: []
    exception exit: {#Ref<0.1026865991.1686372353.206368>,350851,
                     [{child1,<0.302450.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302441.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11906
  neighbours:


</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Thu Jun 13 2024 16:22:36<!--/date--><br />
</div>
</center>
</body>
</html>
